src/components/alert/alert/alert.scss

```diff
@@ -33,9 +35,8 @@ $open-anim-opacity-to: 1;
   }
 }
 
-:host([disable-animation]) {
-  // To avoid flickering, we need 0 instead of 0.1ms here.
-  --sbb-alert-animation-duration: 0ms;
+:host(.sbb-animate) {
```

Comment:
I would also be for using a property because I feel this feature is part of the component API

Reply:
Sorry for flip-flopping. After putting some additional thought into this and talking about it with Jeri, it might be better to have an opt-in with a property (e.g. `animate`).
Thoughts?

Reply:
Consider that for the `sbb-notification`, the property would only affect the opening. Isn't it a bit confusing?

Gut feeling, I would not try to force a common name for it. 
Maybe just define a common pattern for it (e.g. `animate-...`, in this case, `animate-opening`)

Reply:
TBD